src/main/java/com/google/sps/servlets/TripServlet.java

```diff
@@ -89,51 +175,86 @@ private void eventDoGet(HttpServletResponse response) throws IOException {
     }   
 
     response.getWriter().println(convertToJson(events));
-   }
+  }
 
   /**
    * Make the servlet cleaner.
    * Searches through the parameters and creates the events and puts them into
    * datastore.
    */
-  private void eventDoPost(HttpServletRequest request, HttpServletResponse response) 
+  private void eventDoPost(HttpServletRequest request, HttpServletResponse response,
+                          String date, String[] pois) 
       throws IOException { 
     
-    // Print out params to site to verify retrieval of "start trip" user input.
-    Enumeration<String> params = request.getParameterNames();
-
-    // get date of trip
-    String date = request.getParameter("inputDayOfTravel");
-
     // set startDateTime
     LocalDateTime startDateTime = LocalDateTime.of(LocalDate.parse(date), LocalTime.of(10, 0));
 
-    // search through all the parameters looking for pois
-    while (params.hasMoreElements()) {
-      String p = params.nextElement();
-
-      /** 
-       * for each poi create the necessary fields, this will change
-       * as we are able to pull from the maps backend api
-       */
-      if (p.contains("poi")) {
-        String address = request.getParameter(p);
-        String name = address.split(",")[0];
-        Event event = new Event(name, address, startDateTime, HALF_HOUR);
-        Entity eventEntity = event.eventToEntity();
-
-        // put entity in datastore
-        DatastoreService datastore = 
-                                DatastoreServiceFactory.getDatastoreService();      
-        datastore.put(eventEntity);
-
-        // sets start time for next event 2 hours after start of prev
-        startDateTime = startDateTime.plusMinutes(Long.valueOf(NINETY_MINS));
-      }
+    // for each poi create the necessary fields
+    for (String address : pois) {
+      String name = address.split(",")[0];
+      Event event = new Event(name, address, startDateTime, HALF_HOUR);
+      Entity eventEntity = event.eventToEntity();
+
+      // put entity in datastore
+      DatastoreService datastore = 
+                              DatastoreServiceFactory.getDatastoreService();      
+      datastore.put(eventEntity);
+
+      // sets start time for next event 2 hours after start of prev
+      startDateTime = startDateTime.plusMinutes(Long.valueOf(NINETY_MINS));
+    }
+
+    // // redirect to home page
+    // response.sendRedirect("/");
```

Comment:
fix formatting. Either uncomment out or remove whatever you think. Perhaps redirect to /trips/ ?